### DISCUSSION TITLE
Login credentials for frigate web ui?

### DISCUSSION THREAD

**Discussion by hugalafutro** [blakeblackshear/frigate#2479] [Ask A Question] [answered]
Hi,
I went through the docs as I've setup frigate with ha and the frigate card, but I didn't see an option to enable login or password for the frigate web ui running on port 5000. Is it possible to set a password so that page is not accessible to everyone with access to local network ?

Thanks.

**Comment by blakeblackshear** [comment, marked as answer]
There is no auth functionality builtin. Most people access it with ingress behind HA auth. Can you disable the port 5000 access for the addon?

**Reply by hugalafutro** [reply]
I can disable port 5000 by not exposing it in the frigate's docker-compose, but I thought it needs to be open for the HA integration? I run both ha and frigate in separate containers, I guess I could run them both in one stack and not expose the port to outside network, but I would like to retain to access the webui. 

I don't mind folks from downstairs viewing the outside cameras, I would like to restrict view of private rooms cameras though. Or would you suggest forgoing using the webui and create users with access to different lovelace dashboards with different cameras and let folks access ha instead of frigate?

**Reply by blakeblackshear** [reply]
There isnt any way to limit users to cameras in frigate. You can add auth by setting up a proxy that supports auth, but that won't get you per user permissions.

Your best bet is using HA users as you described or running multiple instances of frigate.

**Reply by hugalafutro** [reply]
I haven't thought about running multiple instances, I presume running 1 frigate with 4 cameras or running 2 frigates with 2 cameras each would consume roughly same system resources?

**Reply by blakeblackshear** [reply]
Yes, but keep in mind that you will need a Coral for each instance of frigate. They cannot share.
